pages/contact/index.html

```diff
@@ -82,7 +82,7 @@ <h4>00 (958) 9865 562</h4>
             <p>Mon to Fri 9am to 6 pm</p>
           </div>
         </div class>
```

Comment:
kas čia ?

Reply:
nereikia to class

Reply:
tai kad pries tai buvo texterea uzdetas ir max-heightas ir kas tik nori, sunku daryti, kai nematau rezultato ka padarau